notebooks/examples/configure_check_conditions.ipynb

```diff
@@ -0,0 +1,470 @@
+{
```

Comment:
<p>After this add a note explaining that the condition logic will run and be viewed only inside a Suite, in the Conditions Summary table. For an example for how to do that see the last part in the Add a Custom Check [Link] section.</p><p> </p><p>--</p><p> </p><p><strong>Add a condition to a check in a suite</strong></p><p> </p><p>--</p><p>Add cell:</p><p> </p><p>If we want to add a conditon to a check within an existing suite, we should first find the Check's ID within the suite, and then add the condition to it, by running the relevant <code>add_condition_x</code> method on that check's instance. See the next section to understand how to do so.</p><p>The condition will then be appended to the list of conditions on that check (or be the first one if no conditions are defined), and each condition will be evaluated separately when running the suite.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepchecks/deepchecks/pull/233/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>